Packs/ThycoticDSV/Integrations/ThycoticDSV/ThycoticDSV.py

```diff
@@ -0,0 +1,92 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+#   from typing import Dict
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    def __init__(self, server_url: str, client_id: str, client_secret: str, proxy: bool, verify: bool):
+        super().__init__(base_url=server_url, proxy=proxy, verify=verify)
+        self._client_id = client_id
+        self._client_secret = client_secret
+        self._token = self._generate_token()
+        self._headers = {'Authorization': self._token, 'Content-Type': 'application/json'}
+
+    def _generate_token(self) -> str:
+        body = {
+            "client_id": self._client_id,
+            "client_secret": self._client_secret,
+            "grant_type": "client_credentials",
+            "provider": "thy-one"
```

Comment:
Can you please explain this line?

Reply:
@guykeller 
It seems to have corrected all the descriptions.

Reply:
These parameters are required to perform a REST API call to obtain a new  authentication token.
[rest api](https://dsv.thycotic.com/api#operation/tokenl)

Reply:
I'm not seeing any reference for "thy-one" in the API, can you please tell me exactly where it is?